### PR TITLE
[instancer] Apply cvar deltas 

### PR DESCRIPTION
**Pull request by qxliu76**

### CODE REVIEW

**Comment by behdad** [comment]
Any idea why one bot is failing?

File: src/hb-ot-var-cvar-table.hh [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        if (unlikely (idx >= num_cvt_item)) continue;
+        if (scalar != 1.0f) cvt_deltas[idx] += unpacked_deltas[i] * scalar ;
+        else cvt_deltas[idx] += unpacked_deltas[i];
+      }
+    } while (iterator.move_to_next ());
+
+    return true;
+  }
+
+  bool add_cvt_and_apply_deltas (hb_subset_plan_t *plan) const
```
Comment (behdad) [inline src/hb-ot-var-cvar-table.hh:104]: Should this be a static function?

Comment (qxliu76) [reply src/hb-ot-var-cvar-table.hh]: Done

**Comment by qxliu76** [comment]
> Any idea why one bot is failing?

No..I'm trying to get access and debug on windows machine. Would you have any thoughts on what might be the issue on windows machine?

**Comment by behdad** [comment]
> Would you have any thoughts on what might be the issue on windows machine?

I don't.

**Pull request merged**